Sources/GRPC/ClientCalls/BidirectionalStreamingCall.swift

```diff
@@ -22,63 +22,60 @@ import NIOHTTP2
 ///
 /// Messages should be sent via the `sendMessage` and `sendMessages` methods; the stream of messages
 /// must be terminated by calling `sendEnd` to indicate the final message has been sent.
-public final class BidirectionalStreamingCall<
+public struct BidirectionalStreamingCall<
```

Comment:
This struct still has reference semantics. It'd be good to clarify that in the doc comment. The same applies to the others.

Reply:
Ah, yes, that's certainly worth documenting.